code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -347,6 +346,9 @@ GLOBAL_VAR(bomb_set)
 /obj/machinery/nuclearbomb/blob_act(obj/structure/blob/B)
 	if(timing == -1.0)
 		return
+	if(timing)	//boom
+		INVOKE_ASYNC(src, .proc/explode)
```

Comment:
Will this still run if the object is qdel()ed?

Reply:
It won't work correctly if the object gets qdeled. That's why I have it return early so the nuke isn't deleted.

Reply:
Yeah, fair.